analysis/03_ECMWF_MARS_Vs_CDS.qmd

```diff
@@ -0,0 +1,134 @@
+
+## Overview/Intro
+
+During the initial design phase of the CADC trigger the public CD API was used to access ECMWF SEAS51 data for historical analysis/trigger design. Later, data was downloaded via the MARS API using our WMO access. Since the historical analysis is based on monthly rainfall sums (aggregated by season), this document explores the comparability of MARS ECMWF data vs CDs ECMWF data. A quick overview of ECMWF Seasonal forecast products used in this analysis
+
+-   CDs - SEAS51
+    -   resolution: 1 degree
+    -   leadtimes: 6 available. Leadtime 1 = publication date
+    -   access: public
+    -   considered formats: 51 ensemble member monthly means, monthly mean of ensemble members
+    -   publication date:
+    -   use: initial historical analysis/trigger design
+-   MARS - SEAS51
+    -   Resolution: 0.4 degree
+    -   Leadtimes: 7 available. Leadtime 1 = publication date
+    -   Access: WMO
+    -   Considered formats: 51 ensemble member monthly means, monthly mean of ensemble members
+    -   Publication date:
+    -   Use: evaluation in this document
+
+```{r}
+library(targets)
+library(gghdx)
+library(tidyverse)
+gghdx()
+
+# tar_load(df_ecmwf_mars)
+# tar_load(df_ecmwf_zonal_all) 
+tar_load(df_ecmwf_mars)
+tar_load(df_ecmwf_cds)
+
+ldf_ecmwf <- set_names(
+  list(df_ecmwf_cds, df_ecmwf_mars),
+  c("CDS","MARS")
+  )
+
+# bind CDs & MARS ECMWF data frames
+df_ecmwf <- imap(ldf_ecmwf,\(dft,nm_tmp){
+  dft %>% 
+    mutate(
+      dl_source = nm_tmp
+    ) %>% 
+    filter(stat=="mean")
+}) %>% 
+  list_rbind()
+
+adm_iter <- unique(df_ecmwf$adm0_es)  
+lps <- map(set_names(adm_iter,adm_iter),
+           ~df_ecmwf %>% 
+             # filter(lt!=6,adm0_es==.x) %>% 
+             mutate(
+               decade =year(pub_date) - year(pub_date) %% 10,
+               decade_abbr = paste0(decade,"'s")
+             ) %>% 
+             ggplot(
+               aes(x= valid_date,y= value, color= dl_source)
+             )+
+             geom_point(size=1, alpha=0.5)+
+             geom_line(alpha=0.5)+
+             scale_color_manual(values = c("MARS"=hdx_hex("tomato-hdx"),"CDS"=hdx_hex("sapphire-hdx")))+
+             labs(title = "MARS VS CDS: Average Precip Forecast by decade and leadtime",
+                  subtitle= .x, y= "mm")+
+             scale_x_date(date_breaks= "1 year", date_labels  = "%y")+
+             facet_grid(rows =vars(lt) , cols = vars(decade_abbr),scales="free")+
+             theme(
+               axis.text.x = element_text(angle = 90),
+               legend.title = element_blank()
+             )
+)
+```
+
+```{r}
+lps$Guatemala
+```
+
+```{r}
+lps$Honduras
+```
+
+```{r}
+lps$`El Salvador`
+```
+
+```{r}
+lps$`Nicaragua`
+```
+
+# Quick Correlation Experiment
+
+```{r}
```

Comment:
I think the easiest thing to show is just that you would historically have triggered in the exact same years historically, which also would make people more confident in the trigger, as correlation is more abstract.

Reply:
yeah thats a good idea - im going to pull this into an issue to address later and merge to main to simplify my next git sequencing